static/temp/foods.json

```diff
@@ -0,0 +1,310 @@
+          "Fiber",
+          "Minerals"
+        ],
+        "valuePercent": "% Daily Value",
+        "rdi": {
+          "amount": 130,
+          "unit": "g"
+        }
+      },
+      "tableContent": [
+        {
+          "title": "Total Carbohydrate",
+          "amount": 4,
+          "amountUnit": "g",
+          "content": [
+            {
+              "nutrient": "Carbohydrate  2g",
+              "value": "17.67%",
+              "rdi": {
+                "amount": "130",
+                "unit": "g"
+              }
+            },
+            {
+              "nutrient": "Sugar  2g",
+              "valuePercent": "17.67%",
+              "rdi": { "amount": "15", "unit": "g" }
+            },
+            {
+              "nutrient": "Starch  2g",
+              "value": "17.67%",
+              "rdi": {
+                "amount": "12.6",
+                "unit": "g"
+              }
+            }
+          ]
+        },
+        {
+          "title": "Protein",
+          "amount": 11,
+          "amountUnit": "g",
+          "content": [{ "nutrient": "Protein 2g", "value": "1.61%", "rdi": "46g" }]
+        }
+      ]
+    },
+    "recipes": [
+      {
+        "name": "Banana Walnut Bread",
+        "description": "Flour, baking soda, ripe banana, vanilla extract",
+        "duration": {
+          "prepTime": {
+            "hours": 0,
+            "minutes": 10
+          },
+          "cookTime": {
+            "hours": 0,
+            "minutes": 20
+          },
+          "totalTime": {
+            "hours": 0,
+            "minutes": 30
+          }
+        },
+        "instructions": [
+          {
+            "step": 1,
+            "description": "Wash and cut your banana inside a neat bowl and mix thoroughly till it’s smoothly combined",
+            "videoUrl": ""
+          },
+          {
+            "step": 2,
+            "description": "Dice the spinach into smaller pieces and pour inside your mixed banana and stir thoroughly.",
+            "waitTime": {
+              "hours": 0,
+              "minutes": 10
+            }
+          }
+        ],
+        "reviews": [{ "1": 5, "2": 5, "3": 5, "4": 5, "5": 4 }],
+        "ingredients": [
+          {
+            "name": "Banana",
+            "quantity": 250,
+            "unit": "g"
+          },
+          {
+            "name": "Spinach",
+            "quantity": 100,
+            "unit": "ml"
+          },
+          {
+            "name": "Almond milk ",
+            "quantity": 120,
+            "unit": "ml"
+          },
+          {
+            "name": "Cocoa powder",
+            "quantity": 10,
+            "unit": "g"
+          }
+        ],
+        "nutrition": [],
+        "tags": ["gluten-free", "vegan"]
+      }
+    ],
+    "groceryLists": [
+      {
+        "name": "Banana Smoothie",
+        "image": "/grocery-img",
+        "ingredients": [
+          {
+            "name": "Banana",
+            "quantity": "1"
```

Comment:
add another key in all of these json called `unit`. For example,

```json
[{
  "name": "Banana",
  "quantity": "1",
  "unit": "count"
},
  {
    "name": "Spinach",
    "quantity": "1/2",
    "unit": "cup"
  }]
```

Reply:
done